src/622/modules/password/index.js

```diff
@@ -0,0 +1,10 @@
+const words = ['acesa', 'ajuda', 'amigo', 'antes', 'arcos', 'baile', 'balas', 'bispo', 'chefe','cheio', 'cinto', 'cravo', 'etapa', 'etnia', 'flora', 'lazer', 'legal', 'lugar', 'parte', 'parto', 'perto', 'porta', 'regra', 'resto', 'salve', 'sente', 'setor', 'sexta', 'tecla', 'tinta', 'torta', 'touro', 'trato', 'valer', 'veado']
+
```

Comment:
não esquecer de documentar

```suggestion

/**
 * @todo Documentar
 */
```

Reply:
Documentação adicionada 3236f22585096c181aca40fcd93c271f4d7cad42